test/references/octant_ghost.txt

```diff
@@ -0,0 +1,4 @@
+𜺠𜴯𜷥█𜴴𜷞𜺣
+𜷡🯦𜷕𜷤🯦𜷕▙
+█𜶫█🮅█𜵰█
```

Comment:
These don't seem right. Block and braile have the same detail just different sizes, and are both complete representations of the array they're made from. But this has much less detail than the array does, so it's lossy

Reply:
Your font doesn't have octant characters. You are seeing some of the characters (the all-white and all-black ones), but it seems that the missing characters are replaced by spaces. I see a "missing character" symbol instead when I use the wrong font.

Reply:
<img width="120" alt="pac-octant-missing" src="https://github.com/user-attachments/assets/ce349a41-f3a5-44f2-8e6e-e3715c4e12f3" />
This is how missing characters look for me. Not ideal, but it's obvious that something is going wrong.